src/libraries/System.Security.Cryptography.Primitives/tests/CryptoStream.cs

```diff
@@ -28,6 +28,11 @@ protected override Task<StreamPair> CreateWrappedConnectedStreamsAsync(StreamPai
 
         protected override Type UnsupportedConcurrentExceptionType => null;
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/45080")]
+        [Theory]
+        [MemberData(nameof(ReadWrite_Success_Large_MemberData))]
```

Comment:
```suggestion
```

Reply:
This makes XUnit complain: `error xUnit1013: Public method 'ReadWrite_Success_Large' on test class 'ConnectedStreamConformanceTests' should be marked as a Theory.`

Interestingly, XUnit is actually complaining about the wrong class.

Reply:
We do the same thing elsewhere, e.g.
https://github.com/dotnet/runtime/blob/bf302de7f184c6e9dc23382980860af2849219e3/src/libraries/System.Net.Quic/tests/FunctionalTests/QuicStreamConnectedStreamConformanceTests.cs#L24-L36
I would need to try it locally to know why this is different.

Reply:
Huh. Strange.